### PR TITLE
XSA/SHINE - Complete e2e install and config; output is URL for SHINE-UI

### PR DESCRIPTION
**Pull request by tniek**
Additional variables (mainly XSA/SHINE bits and passwords) are required
in TF to be passed through to Ansible:
- url_xsa_runtime
- url_di_core
- url_sapui5
- url_portal_services
- url_xs_services
- url_shine_xsa
- pwd_db_xsaadmin
- pwd_db_tenant
- pwd_db_shine
- email_shine

### CODE REVIEW

File: experiment/ansible/roles/xsa-install/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  command: ./SAPCAR_LINUX.EXE -manifest SIGNATURE.SMF -xvf EXTAPPSER00P_87-70001316.SAR -R EXTAPPSER00P_87
+  args:
+    chdir: /hana/shared/install
+    creates: /hana/shared/install/EXTAPPSER00P_87
+
+- name: Run hdblcm to install XSA components
+  shell: "pwd=$(<hdblcm_xsa.passwords); rm hdblcm_xsa.passwords; echo $pwd | SAP_HANA_DATABASE/hdblcm --component_root=/hana/shared/install/EXTAPPSER00P_87 --configfile=hdblcm_xsa.cfg --batch --read_password_from_stdin=xml"
+  args:
+    chdir: /hana/shared/install
+
```
Comment (pabowers) [inline experiment/ansible/roles/xsa-install/tasks/main.yml]: nit: newline

**Comment by persiaAziz** [comment]
Are the empty files supposed to be empty?

File: experiment/ansible/roles/shine-install/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+# This role installs and configures SHINE on top of an existing XSA instance
+- name: Download SHINE Content (XSA)
+  get_url:
+    url: "{{ url_shine_xsa }}"
+    dest: /usr/sap/{{ sap_sid|upper }}/HDB{{ sap_instancenum }}/XSASHINE.ZIP
+
+- name: Log into XSA
+  shell: |
+    . ~/.bashrc
+    xs login --skip-ssl-validation -a https://{{ ansible_fqdn }}:3{{ sap_instancenum }}30/ -s SAP -u XSA_ADMIN  -p {{ pwd_db_xsaadmin }}
```
Comment (persiaAziz) [inline experiment/ansible/roles/shine-install/tasks/main.yml]: is xs already installed?

Comment (tniek) [reply experiment/ansible/roles/shine-install/tasks/main.yml]: Yes; the shine-install role gets run in the Ansible playbook after xs-install.

**Pull request merged**